client/public/locales/en/translation.json

```
@@ -51,8 +51,11 @@
     "override": "Override",
     "reset": "Reset",
     "review": "Review",
+    "retrieve": "Retrieve",
+    "retrieveConfigurations": "Retrieve configurations",
```

Biome (2.1.2)

[error] The key retrieveConfigurations was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)